### PR TITLE
Fix LineLength Check Suppressions: index.fielddata

### PR DESCRIPTION
**Pull request by original-brownbear**
* Fix linelength suppressions in index.fielddata
  * Some lines that were too long were dead code => Removed them and all code that became dead because of it
* Relates #34884

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/index/fielddata/IndexFieldDataCache.java [hunk 1]
```diff
@@ -59,13 +60,15 @@ default void onRemoval(ShardId shardId, String fieldName, boolean wasEvicted, lo
...
+        public <FD extends AtomicFieldData, IFD extends IndexFieldData<FD>> FD load(LeafReaderContext context, IFD indexFieldData)
+            throws Exception {
             return indexFieldData.loadDirect(context);
         }
 
         @Override
         @SuppressWarnings("unchecked")
-        public <FD extends AtomicFieldData, IFD extends IndexFieldData.Global<FD>> IFD load(DirectoryReader indexReader, IFD indexFieldData) throws Exception {
+        public <FD extends AtomicFieldData, IFD extends IndexFieldData.Global<FD>> IFD load(DirectoryReader indexReader,
+            IFD indexFieldData) throws Exception {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/IndexFieldDataCache.java]: Could you indent this one more time so the method declaration doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/DoubleValuesComparatorSource.java [hunk 2]
```diff
@@ -43,7 +43,8 @@
 
     private final IndexNumericFieldData indexFieldData;
 
-    public DoubleValuesComparatorSource(IndexNumericFieldData indexFieldData, @Nullable Object missingValue, MultiValueMode sortMode, Nested nested) {
+    public DoubleValuesComparatorSource(IndexNumericFieldData indexFieldData, @Nullable Object missingValue, MultiValueMode sortMode,
+        Nested nested) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/DoubleValuesComparatorSource.java]: Could you indent this one more time so the method declaration doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/FloatValuesComparatorSource.java [hunk 3]
```diff
@@ -41,7 +41,8 @@
 
     private final IndexNumericFieldData indexFieldData;
 
-    public FloatValuesComparatorSource(IndexNumericFieldData indexFieldData, @Nullable Object missingValue, MultiValueMode sortMode, Nested nested) {
+    public FloatValuesComparatorSource(IndexNumericFieldData indexFieldData, @Nullable Object missingValue, MultiValueMode sortMode,
+        Nested nested) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/FloatValuesComparatorSource.java]: Same here.

File: server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/LongValuesComparatorSource.java [hunk 4]
```diff
@@ -40,7 +40,8 @@
 
     private final IndexNumericFieldData indexFieldData;
 
-    public LongValuesComparatorSource(IndexNumericFieldData indexFieldData, @Nullable Object missingValue, MultiValueMode sortMode, Nested nested) {
+    public LongValuesComparatorSource(IndexNumericFieldData indexFieldData, @Nullable Object missingValue, MultiValueMode sortMode,
+        Nested nested) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/LongValuesComparatorSource.java]: Same here.

File: server/src/main/java/org/elasticsearch/index/fielddata/ordinals/GlobalOrdinalsBuilder.java [hunk 5]
```diff
@@ -79,7 +79,8 @@ public static IndexOrdinalsFieldData build(final IndexReader indexReader, IndexO
         );
     }
 
-    public static IndexOrdinalsFieldData buildEmpty(IndexSettings indexSettings, final IndexReader indexReader, IndexOrdinalsFieldData indexFieldData) throws IOException {
+    public static IndexOrdinalsFieldData buildEmpty(IndexSettings indexSettings, final IndexReader indexReader,
+        IndexOrdinalsFieldData indexFieldData) throws IOException {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/ordinals/GlobalOrdinalsBuilder.java]: Same here.

File: server/src/main/java/org/elasticsearch/index/fielddata/ordinals/MultiOrdinals.java [hunk 6]
```diff
@@ -46,7 +46,8 @@
     /**
      * Return true if this impl is going to be smaller than {@link SinglePackedOrdinals} by at least 20%.
      */
-    public static boolean significantlySmallerThanSinglePackedOrdinals(int maxDoc, int numDocsWithValue, long numOrds, float acceptableOverheadRatio) {
+    public static boolean significantlySmallerThanSinglePackedOrdinals(int maxDoc, int numDocsWithValue, long numOrds,
+        float acceptableOverheadRatio) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/ordinals/MultiOrdinals.java]: Same here.

File: server/src/main/java/org/elasticsearch/index/fielddata/plain/AbstractIndexOrdinalsFieldData.java [hunk 7]
```diff
@@ -36,7 +36,8 @@
 
 import java.io.IOException;
 
-public abstract class AbstractIndexOrdinalsFieldData extends AbstractIndexFieldData<AtomicOrdinalsFieldData> implements IndexOrdinalsFieldData {
+public abstract class AbstractIndexOrdinalsFieldData extends AbstractIndexFieldData<AtomicOrdinalsFieldData>
+    implements IndexOrdinalsFieldData {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/plain/AbstractIndexOrdinalsFieldData.java]: Could you indent this one more time so it doesn't line up with the class body?

File: server/src/main/java/org/elasticsearch/index/fielddata/plain/AbstractLatLonPointDVIndexFieldData.java [hunk 8]
```diff
@@ -43,7 +43,8 @@
     }
 
     @Override
-    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested, boolean reverse) {
+    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested,
+        boolean reverse) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/plain/AbstractLatLonPointDVIndexFieldData.java]: Could you indent this one more time so it doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/index/fielddata/plain/BinaryDVIndexFieldData.java [hunk 9]
```diff
@@ -46,7 +46,8 @@ public BinaryDVAtomicFieldData loadDirect(LeafReaderContext context) throws Exce
     }
 
     @Override
-    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested, boolean reverse) {
+    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested,
+        boolean reverse) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/plain/BinaryDVIndexFieldData.java]: Could you indent this one more time so it doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/index/fielddata/plain/PagedBytesIndexFieldData.java [hunk 10]
```diff
@@ -65,7 +65,7 @@ public Builder(double minFrequency, double maxFrequency, int minSegmentSize) {
 
         @Override
         public IndexOrdinalsFieldData build(IndexSettings indexSettings, MappedFieldType fieldType,
-                                                               IndexFieldDataCache cache, CircuitBreakerService breakerService, MapperService mapperService) {
+            IndexFieldDataCache cache, CircuitBreakerService breakerService, MapperService mapperService) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/plain/PagedBytesIndexFieldData.java]: Could you indent this one more time so it doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/index/fielddata/plain/PagedBytesIndexFieldData.java [hunk 11]
```diff
@@ -78,7 +78,8 @@ public PagedBytesIndexFieldData(IndexSettings indexSettings, String fieldName,
     }
 
     @Override
-    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested, boolean reverse) {
+    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested,
+        boolean reverse) {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/index/fielddata/plain/PagedBytesIndexFieldData.java]: Could you indent this one more time so it doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/indices/fielddata/cache/IndicesFieldDataCache.java [hunk 12]
```diff
@@ -125,7 +128,8 @@ public long applyAsLong(Key key, Accountable ramUsage) {
         }
 
         @Override
-        public <FD extends AtomicFieldData, IFD extends IndexFieldData<FD>> FD load(final LeafReaderContext context, final IFD indexFieldData) throws Exception {
+        public <FD extends AtomicFieldData, IFD extends IndexFieldData<FD>> FD load(final LeafReaderContext context,
+            final IFD indexFieldData) throws Exception {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/indices/fielddata/cache/IndicesFieldDataCache.java]: Could you indent this one more time so it doesn't line up with the method body?

File: server/src/main/java/org/elasticsearch/indices/fielddata/cache/IndicesFieldDataCache.java [hunk 13]
```diff
@@ -151,7 +155,8 @@ public long applyAsLong(Key key, Accountable ramUsage) {
         }
 
         @Override
-        public <FD extends AtomicFieldData, IFD extends IndexFieldData.Global<FD>> IFD load(final DirectoryReader indexReader, final IFD indexFieldData) throws Exception {
+        public <FD extends AtomicFieldData, IFD extends IndexFieldData.Global<FD>> IFD load(final DirectoryReader indexReader,
+            final IFD indexFieldData) throws Exception {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/indices/fielddata/cache/IndicesFieldDataCache.java]: Could you indent this one more time so it doesn't line up with the method body?

**Review by nik9000 — APPROVED** [review]
I left a baker's dozen requests to add a single extra level of indentation.

**Comment by original-brownbear** [comment]
Jenkins test this

**Comment by original-brownbear** [comment]
@nik9000 thanks!

**Pull request merged**